### PR TITLE
Portfolio header

### PR DESCRIPTION
**Pull request by phelian23**
Create my portfolio header HTML and CSS

### CODE REVIEW

**Comment by addod19** [comment]
### Hi @phelian23, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that I could not review because we have a failing linter and the README file is not professional or descriptive

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by phelian23** [comment]
Hi @addod19,

Thank you for the review. The errors have been fixed, and I've updated the README.md

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="styles.css" rel="stylesheet"><title>My Portfolio</title>
+    </head>
+    <body>
+        <header>
+            <p class="logo">Femi</p>
+            <a class="togglebar" href="#"><img src="Images/Icon-Menu.png" alt="menu icon"></a>
```
Comment (afanimartin) [inline index.html:11]: It's a good idea to use semantic elements on the page when ever possible. Please wrap the hamburger icon with a nav element.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                <li><a href="#">portfolio</a></li>
+                <li><a href="#">about</a></li>
+                <li><a href="#">contact</a></li>
+                <li><a href="#"><img src="Images/Icon-Mail.png" alt="mail icon"></a></li>
+            </ul>
+        </header>
+        <section class="headline">
+            <div class="section-text">
+                <h1 class="headline1">Hey There.<br>I'm Femi</h1>
+                <h3 class="headline2">I'am a Software Developer</h3>
```
Comment (afanimartin) [inline index.html]: Please change this to a p tag. H3 is too big for this content.

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @phelian23, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- You have a horizontal scrollbar on the page. This is supposed to be a mobile design and so contents should be properly rendered on the page. Please fix the scrollbar.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._,

File: index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="styles.css" rel="stylesheet"><title>My Portfolio</title>
+    </head>
+    <body>
+        <nav>
+            <p class="logo">Femi</p>
```
Comment (SajjadAhmad14) [inline index.html:10]: [  OPTIONAL ] Please wrap the logo in a link tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                <p class="headline2">I'am a Software Developer</p>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            </div>
+            <ul class="container">
+                <li><img src="Images/github.png" alt="github"></li>
+                <li><img src="Images/Linkedinicon.png" alt="linkedin"></li>
+                <li><img src="Images/angellist.png" alt="Angel list"></li>
+                <li><img src="Images/twitter.png" alt="Twitter"></li>
+                <li><img src="Images/medium.png" alt="Medium"></li>
```
Comment (SajjadAhmad14) [inline index.html:31]: [ OPTIONAL ] These icons should be clickable links.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,42 @@
-# My-portfolio
-Creating a website for my portfolio
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Portfolio
+
+> Creating a website for my portfolio.
+
+![screenshot](./app_screenshot.png)
```
Comment (SajjadAhmad14) [inline README.md:7]: [ OPTIONAL ] Screenshot image is broken. Please fix that.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
### Hi Femi Awoyemi, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
# OPTIONAL BUT RECOMMENDED.
- [ ] Please see inline comments.
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**